.doc_gen/readmes/README.md

```diff
@@ -0,0 +1,103 @@
+# WRITEME: auto-generated READMEs
+
+## Overview
+
+Describes WRITEME, a tool to automatically generate service-level READMEs from
+metadata and Jinja templates.
+
+This is an internal tool intended for use only by the AWS code examples team.
+
+## Prerequisites
+
+You must have a recent version of Python installed to run this tool, 
+and a recent version of pip (Python's package manager) to install the
+required packages.
+
+### Install packages
+
+A virtual environment is recommended. Create a virtual environment 
+and install packages by running the following commands in the
+`.doc_gen/readmes` folder:
+
+```
+python -m venv venv
+venv/Scripts/activate
+python -m pip install -r requirements.txt
+```
+
+Depending on how you have Python installed and on your operating system,
+the commands may vary slightly, for example on Windows use `py` in place of
+`python` and uses backslashes in the `venv` path.
```

Comment:
Depending on how you have Python installed and on your operating system,
the commands might vary slightly. For example, on Windows, use `py` in place of
`python` and use backslashes in the `venv` path.

Reply:
fixed